src/users/src/users-service/repository.py

```diff
@@ -0,0 +1,111 @@
+import random
+from models import User, Address
+import json
+import gzip
+from pynamodb.exceptions import DoesNotExist
+
+claimed_users = {}
+
+def init():
+    try:
+        load_users_into_dynamodb("/src/data/users.json.gz")
+        return True
+    except Exception as e:
+        raise e
+    
+def load_users_into_dynamodb(filename):
+    with gzip.open(filename, 'rt', encoding='utf-8') as file:
+        data = json.load(file)
+        for user_data in data:
+            create_or_update_user(user_data)
+
+def create_or_update_user(user_data):
+    try:
+        user = User.get(user_data["id"])
+        print(f"User already exists: {user_data['id']}")
+        #update_user_with_data(user, user_data)
+    except DoesNotExist:
+        print(f"Creating new user: {user_data['id']}")
+        user = User()
+        update_user_with_data(user, user_data)
+        user.save()
+
+def update_user_with_data(user, user_data):
+    print(f"Updating user:{user} with data: {user_data}")
+    addresses_data = user_data.pop("addresses", [])
+    addresses = [Address(**ad) for ad in addresses_data]
+    user.addresses = addresses
+    for key, value in user_data.items():
+        setattr(user, key, value)
+    user.save()
+
+def get_all_users():
+    return User.scan()
+
+def get_user_by_id(user_id):
+    user = User.get(user_id)
+    if user:
+            return user
+    else:
+        return User()
+
+def get_user_by_username(username):
+    try:
+        user = next(User.username_index.query(username), None)
```

Comment:
Can you just return directly from this statement?

Reply:
yeah can return from this replacing None with User(). again looks like similar leftovers from default change. I'll run through everything to see if I it appears elsewhere